lib/plugin.lazy.js

```diff
@@ -8,6 +8,18 @@ let SentryMock = {}
 let sentryReadyResolve
 let loaded = false
 
+<% if (options.lazy.injectMock) { %>
+const vueErrorHandler = VueLib.config.errorHandler
+
+VueLib.config.errorHandler = (err, vm, info) => {
+  vm.$sentry.captureException(err)
+
+  if (vueErrorHandler) {
+    return vueErrorHandler(err, vm, info)
+  }
+}
+<% } %>
```

Comment:
Bbased on my previous reading of Sentry code, I think that will be problematic because when sentry is loaded, it will wrap our handler with its own, and then `captureException` will be triggered twice, once from here and once from Sentry.

We would need to restore state of `errorHandler` to what we started with before Sentry is loaded.

Reply:
Thinking more about it, due to the nature of the global handler, I think it would be risky to try to unset our handler as it might have been changed by something else before Sentry does it.

A better idea would be to keep the function assigned but just change the inner-workings of it like:
```js
let captureExceptionHandler = (err, vm, info) => {
  vm.$sentry.captureException(err)
}

VueLib.config.errorHandler = (err, vm, info) => {
  captureExceptionHadler(err, vm, info)

  if (vueErrorHandler) {
    return vueErrorHandler(err, vm, info)
  }
}
```

and then, from `loadSentry()` re-assign `captureExceptionHandler` to a no-op function.

Reply:
Addressed with e5dc66434467207665ca115f258028965d233f50

Reply:
@rchl You could wrap the sentry capture call in an if not loaded. Then at least sentry wouldnt be called twice?

Reply:
It seems like using `loaded` would potentially leave some window where we might not catch errors since `loaded` is set at the beginning of `loadSentry` but loading might be delayed. So I think I'll introduce now one more variable for this.

Reply:
Addressed in 4452ac3d86c9e07f11219f59e65d7ec5c7599931